### ISSUE TITLE
Suggestion: Allow the user to set a password if none has been set

### ISSUE THREAD

**Issue by JackDallas** [dragoonDorise/EmuDeck#250]
Currently on a first run of emu deck if no password has been set but the user assumes their steam password will work and inputs that the UI will keep running as if a correct password has been entered while the console is throwing errors about the password needing to be set. At this point the user could be prompted to set the password via the ui or the installer could show an error informing the user they need to set the password and then exit the installer

**Comment by JackDallas** [comment]
I was about to start a PR but, just looked at the code on main and this has been added, I just got my deck so maybe that hasn't been release yet? If it has I may have found a bug, I'll have a look at what version I'm on and get back

**Comment by Godsbane** [comment]
Yes it does check using passwd -S to query the status, and then awk's over the results for the status code. I have adjusted it in the dev / beta channels as well, but it still may very well have an issue.

**Comment by JackDallas** [comment]
Happy to mark this closed and assume I was being dumb, when I inevitable have to reinstall the os on my deck I'll pay more attention when installing emudeck to see if there's actually an issue still
